### PR TITLE
Sign up and login form responsive

### PR DESCRIPTION
**Pull request by sakshikhachane**
## Issue that this pull request solves

 Closes: #182 

### Brief description of what is fixed or changed
Sign up and login form are now not compressed by reducing screen width. adjusted the screen likewise.

## Types of changes

_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] My changes does not break the current system and it passes all the current test cases.

## Screenshots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

File: src/app/components/home/home.component.ts [hunk 1]
```diff
@@ -10,7 +10,7 @@ export class HomeComponent implements OnInit {
 
   constructor() { }
 
-  ngOnInit(){
+  ngOnInit() {
```
Comment (goliakshay357) [inline src/app/components/home/home.component.ts:13]: Are you using Prettier for code formatting?

File: .gitignore [hunk 2]
```diff
@@ -48,4 +48,7 @@ Thumbs.db
 .firebase
```
Comment (goliakshay357) [inline .gitignore:48]: hey, you can just ignore files while pushing. Need not add it in .gitignore and then updating .gitignore.
This might create few problems in the future too.
You can commit only necessary files before making a push. 

Seems like you are committing all the changes
Instead of doing  ```git add .```, commit only the necessary files

File: src/app/components/home/home.component.html [hunk 3]
```diff
@@ -40,12 +40,43 @@
...
+          </div>
+
+          <!-- Left and right controls -->
+          <a class="carousel-control-prev" href="#slider" data-slide="prev">
+            <span class="carousel-control-prev-icon"></span>
+          </a>
+          <a class="carousel-control-next" href="#slider" data-slide="next">
+            <span class="carousel-control-next-icon"></span>
+          </a>
+
```
Comment (goliakshay357) [inline src/app/components/home/home.component.html:78]: Is this code relevant to the issue chosen?

**Comment by Kajol-Kumari** [comment]
@sakshikhachane can you please ping me on slack once. Will have to discuss few stuffs related to the PRs
